src/storage/txn/commands/flush.rs

```diff
@@ -391,4 +412,17 @@ mod tests {
         must_commit(&mut engine, k, 1, 2);
         must_get(&mut engine, k, 3, v);
     }
+
+    #[test]
+    fn test_flushed_existence_check() {
+        let mut engine = TestEngineBuilder::new().build().unwrap();
+        let k = b"key";
+        let v = b"value";
+        must_flush_put(&mut engine, k, *v, k, 1, 1);
```

Comment:
This case checks the existence of flushed lock.
What about adding a cacse which checks the existence of a committed txn's write record?

Reply:
Done